package.json

```diff
@@ -15,7 +15,8 @@
     "packages/SwingSet",
     "packages/ERTP",
     "packages/cosmic-swingset",
-    "packages/agoric-cli"
+    "packages/agoric-cli",
+    "packages/zoe"
```

Comment:
Lets put this between ERTP and cosmic-swingset. I'd like to keep `workspaces` in dependency order, so that if `npm build` in e.g. SwingSet produces a bundle, the subsequent `npm build` in `cosmic-swingset` will pick it up correctly (no "forward references", as it were). I don't know what sort of bundles or build artifacts Zoe might produce, but they should probably appear before `cosmic-swingset`.

Reply:
Good idea